src/lib/style/base/_core.scss

```diff
@@ -260,7 +260,7 @@ a.SRC-whiteText {
 }
 
 .SRC-no-underline-on-hover:hover {
-  text-decoration: none;
+  text-decoration: none !important;
```

Comment:
If we are explicitly adding the SRC-no-underline-on-hover style to an element, I think we mean it.